### PR TITLE
WIP: cloudstack: new module cs_serviceoffer

### PR DESCRIPTION
**Pull request by resmo**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME
cs_serviceoffer

##### ANSIBLE VERSION

```
2.3
```

##### SUMMARY
new module cs_serviceoffer

### CODE REVIEW

File: lib/ansible/modules/cloud/cloudstack/cs_serviceoffer.py [hunk 1]
```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2016, René Moser <mail@renemoser.net>
```
Comment (marcaurele) [inline lib/ansible/modules/cloud/cloudstack/cs_serviceoffer.py:4]: 2017 ;-)

File: lib/ansible/modules/cloud/cloudstack/cs_serviceoffer.py [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+
+    def absent_service_offering(self):
+        service_offering = self.get_service_offering()
+        args = {
+            'id': service_offering['id'],
+        }
+        if service_offering:
+            if not self.module.check_mode:
+                res = self.cs.deleteServiceOffering(**args)
+                if 'errortext' in res:
```
Comment (marcaurele) [inline lib/ansible/modules/cloud/cloudstack/cs_serviceoffer.py:332]: `self.result['changed'] = True` ?

Comment (marcaurele) [reply lib/ansible/modules/cloud/cloudstack/cs_serviceoffer.py:332]: Should even go at line 330, before the check on `check_mode`

**Comment by resmo** [comment]
closed in favor of #33060

**Pull request closed without merging**